addons/account/tests/test_portal_attachment.py

```diff
@@ -100,7 +100,7 @@ def test_01_portal_attachment(self):
         self.assertEqual(res_image.content, b'<svg></svg>')
 
         # Test attachment can't be removed without valid token
-        res = self.opener.post(
+        res = self.url_open(
```

Comment:
Note that most of those tests still using `url_open` with `json` params were defined before `make_jsonrpc_request` existed and can probably be converted to the easier API now.